InstallSpinnaker.sh

```diff
@@ -537,15 +555,26 @@ case $CLOUD_PROVIDER in
       set_google_region
       set_google_zone
       ;;
+  azure)
```

Comment:
Could you fix these so they are orthogonal? Could you refactor it so the provider defaults to none and you can add each independently? That would make it easy to have arbitrary combinations. Then "both" can go away and be replaced by naming aws and google.